server/controllers/mobile-phone-number-change-controller.js

```diff
@@ -14,8 +14,11 @@ const mobilePhoneNumberChange = (req, res) => {
     if (org.slug === reqOrg) {
       // merge default config and custom config
       const conf = merge(defaultConfig, org);
-      const {host} = conf;
-      const url = reverse("mobile_phone_number_change", org.slug);
+      const {host, custom, radiusSlug} = conf;
+      const url = reverse(
+        "mobile_phone_number_change",
+        custom ? radiusSlug : org.slug,
```

Comment:
I see this ternary operator is used in all the sever controller, is there a chance you can find a way to avoid it? For example, by loading this value in a new property and reusing this new property in all these lines?